roles/lobsters/vars/main.yml

```diff
@@ -10,6 +10,6 @@ required_packages:
   - make
   - mariadb-client
   - nodejs
-  - ruby2.3
-  - ruby2.3-dev
+  - ruby2.5
+  - ruby2.5-dev
```

Comment:
This commit I can't put in until production is running Ubuntu 18.  May I ask it to be submitted as a separate PR?  The rest of this work is ansible-related, and I can put in prior to upgrading production.

Reply:
Done.